code/modules/admin/verbs/adminpm.dm

```diff
@@ -47,14 +45,49 @@
 	else if(istype(whom,/client))
 		C = whom
 	if(!C)
-		if(holder)
-			src << "<font color='red'>Error: Admin-PM: Client not found.</font>"
+		if(holder)	src << "<font color='red'>Error: Admin-PM: Client not found.</font>"
 		return
-	message_admins("[key_name_admin(src)] has started replying to [key_name(C, 0, 0)]'s admin help.")
+
+	var/datum/adminticket/ticket
+
+	for(var/datum/adminticket/T in admintickets)

```

Comment:
Please use _istypeless for loops_ you can find more information about this in the [contributing.md](https://github.com/HippieStation/HippieStation13/blob/master/CONTRIBUTING.md)